WordPress/Classes/Utility/BuildInformation/RemoteFeatureFlag.swift

```diff
@@ -62,6 +63,8 @@ enum RemoteFeatureFlag: Int, CaseIterable {
             return AppConfiguration.isJetpack
         case .siteEditorMVP:
             return true
+        case .jetpackSocialImprovements:
+            return AppConfiguration.isJetpack && BuildConfiguration.current == .localDeveloper
```

Comment:
Should we remove `BuildConfiguration.current == .localDeveloper` here? Looks like it'll always be true as long as the app is Jetpack.